server/nodejs/controllers/AliasesService.js

```diff
@@ -186,26 +186,11 @@ exports.aliasupdatewhitelist = function(args, res, next) {
   syscoinClient.aliasUpdateWhitelist.apply(syscoinClient, arr);
 }
 
-exports.aliasnewfund = function(args, res, next) {
-  var argList = [
-    { prop: "hexstring" },
-    { prop: "addresses" }
-  ];
-
-  var cb = function(err, result, resHeaders) {
-    res.setHeader('Content-Type', 'application/json');
-
-    if (err) {
-      return commonUtils.reportError(res, err);
-    }
+exports.syscointransactionfund = methodGenerator.generateGenericSyscoinMethod([
+  { prop: 'hexstring' },
+  { prop: 'addresses' }
+], 'syscointxfund', syscoinClient.syscointxfund, 'POST');
```

Comment:
Note here that the export for the end consumer is syscointransactionfund, while the internal delegation is to the RPC's syscointxfund.